compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -48,8 +48,7 @@
 import java.util.Set;
 import java.util.Stack;
 
-import static org.wso2.ballerinalang.compiler.util.Constants.OPEN_SEALED_ARRAY_INDICATOR;
-import static org.wso2.ballerinalang.compiler.util.Constants.UNSEALED_ARRAY_INDICATOR;
+import static org.wso2.ballerinalang.compiler.util.Constants.*;
```

Comment:
Can you fix this as well? We are not using * for imports.

Reply:
You can configure IDEA to prevent happening this automatically.

Reply:
Sorry :( I missed it. Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/14163/commits/e885b358278c595b5b2cd67515a657e0aa7228b8